src/index.css

```
@@ -19,6 +19,12 @@ main {
   height: 100%;
   overflow-y: auto;
 }
+.plugins-table thead th,
+.scripts-table thead th {
+  position: sticky;
+  top: 0;
+  z-index: 1;
```

I think you can use `z-index` to match Bootstrap's specification.

```suggestion
  z-index: 1030;
```